wiki/Building-From-Source.md

```diff
@@ -44,17 +59,17 @@ cargo xtask prepare-deps --platform [your platform] [--gpl] [--no-nvidia]
 ```
 
 * Replace `[your platform]` with your computer OS, either `windows` or `linux`
-* Use the `--gpl` flag if you want to download, build and bundle FFmpeg inside the ALVR server. Keep in mind that on Windows this is needed only for software encoding; on Linux FFmpeg is always needed, and if you omit the `--gpl` flag, the system ffmpeg libraries will be used, which can cause compatibility issues at runtime. As the name suggest, if you use this flag you can only redistribute the final package as GPLv2.0 licensed (because of the x264 encoder).
-* Use the flag `--no-nvidia` only on Linux and if you have an AMD gpu.
-* This is not required if your Linux distribution includes Vulkan support in its ffmpeg package, like Arch (see above).
+* **Windows only:** Use the `--gpl` flag if you want to download, build and bundle FFmpeg inside the ALVR server. Keep in mind that on Windows this is needed only for software encoding. As the name suggest, if you use this flag you can only redistribute the final package as GPLv2.0 licensed (because of the x264 encoder).
+* Use the flag `--no-nvidia` only on Linux and if you have a AMD gpu.
+* This command is not required if your Linux distribution includes Vulkan support in its ffmpeg package, like Arch (see above).
```

Comment:
As of the latest changes, this command is always required, so delete this line.

Reply:
The "Vulkan support in its ffmpeg package" line?

Reply:
Yes, because now Vulkan is always built from source and statically linked.

Reply:
Done!